### PR TITLE
Delete Poll Option Added

### PR DESCRIPTION
**Pull request by Prateek0129**

### CODE REVIEW

File: src/app/homepage/create/create.component.ts [hunk 1]
```diff
@@ -10,34 +10,52 @@ import { HttpService } from '../../http.service';
...
-    this.createPollFormview = new FormGroup({
-      title: new FormControl('', Validators.required),
-      option1: new FormControl('', Validators.required),
-      option2: new FormControl('', Validators.required),
-      option3: new FormControl('', Validators.required),
-      option4: new FormControl('', Validators.required),
-    });
+    this.createPollFormview = this.fb.group({
+      title: '',
+      option1: undefined,
```
Comment (upexcel) [inline src/app/homepage/create/create.component.ts]: why undefined?

Comment (Prateek0129) [reply src/app/homepage/create/create.component.ts]: there's no particular reason for that.
i can put either a blank(' '), null or undefined.

and if i put blank which is kind of a convention.
the option has no values in the take and view poll.
and if i use undefined or null. 
We get undefined as a default value.
Just Acc. to  MY PREFERENCE.

Comment (Prateek0129) [reply src/app/homepage/create/create.component.ts]: there's no particular reason for that.
i can put either a blank(' '), null or undefined.
Just Acc. to  MY PREFERENCE.

Comment (Prateek0129) [reply src/app/homepage/create/create.component.ts]: I will change it to null. but not blank

Comment (Prateek0129) [reply src/app/homepage/create/create.component.ts]: as null can be seen in view and take poll

**Pull request merged**